package.json

```diff
@@ -72,6 +68,7 @@
         "husky": "^4.3.0",
         "jest": "^26.4.2",
         "lint-staged": "^10.4.0",
+        "npm-run-all": "^4.1.5",
```

Comment:
Doesn't look like you're using this?

Reply:
@hardchor `run-p` command is part of it, will use it to run all scripts for examples (I will add more examples folders in future)